src/encoder/encodersndfileflac.cpp

```diff
@@ -1,11 +1,40 @@
+#include "encoder/encodersndfileflac.h"
+
 #include <QtDebug>
 
-#include "encoder/encodersndfileflac.h"
 #include "encoder/encodercallback.h"
+#include "util/sample.h"
+
+namespace {
+constexpr SINT kEncBufferSize = 8192; // used inside libsndfile for flac
+
+//static
+void convertFloat32ToIntFormat(int* pDest, const CSAMPLE* pSrc, SINT numSamples, int format) {
+    const CSAMPLE kConversionFactor = INT_MIN * -1.0f;
```

Comment:
constexpr? I suggest to extract the constant from this function into the enclosing namespace, e.g. `kFloatToIntConversionFactor`.